mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MailboxMapperTest.java

```diff
@@ -110,35 +108,54 @@ void createShouldThrowWhenMailboxAlreadyExists() throws MailboxException {
 
     @Test
     void createShouldSetAMailboxIdForMailbox() throws MailboxException {
-        benwaInboxMailbox.setMailboxId(null);
         MailboxId mailboxId = mailboxMapper.create(benwaInboxMailbox);
 
         assertThat(mailboxId).isNotNull();
     }
 
     @Test
     void createShouldThrowWhenMailboxIdNotNull() {
+        benwaInboxMailbox.setMailboxId(generateId());
+
         assertThatThrownBy(() -> mailboxMapper.create(benwaInboxMailbox))
```

Comment:
I think here the test subject is more "trying to rename a non existing mailbox"

Because by definition you CAN create a non existing mailbox :trollface:

Reply:
It's a test on checking that create throws if a mailboxId is set up... nothing with rename?